packages/hydrooj/src/handler/judge.ts

```diff
@@ -245,6 +246,15 @@ export class JudgeConnectionHandler extends ConnectionHandler {
         this.send({ language: setting.langs });
     }
 
+    @subscribe('problem/syncData')
```

Comment:
Parallel downloading of all judges can use up all web server bandwidth.
It would be better to let all judges download one by one.